### DISCUSSION TITLE
Disable truncation of title/description in "date" mode (adjustable line-clamp)

### DISCUSSION THREAD

**Discussion by mhaszprunar** [add2cal/add-to-calendar-button#495] [Ideas / Requests]
Hey!
First I got to say that I LOVE your project. Amazing. I however have the problem that the description texts and names are quite long and they always got truncated. 

With an adjustable line-clamp (eg setting from 1 to 2) or the complete disablement of line-clamp (not sure why this is even there to be honest) the button would show all texts as intended.
Would love for you to allow this!

**Comment by jekuer** [comment]
Will add this for the event title

**Comment by jekuer** [comment]
You can now (since v2.5) change this via the styleLight option! 🚀
